src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/importer/report/MoveFilterer.kt

```diff
@@ -29,28 +30,32 @@ object MoveFilterer {
       move.moveDate != null && move.events.find {
       it.hasType(EventType.MOVE_CANCEL)
     }?.occurredAt?.plusHours(9)?.isAfter(move.moveDate?.atStartOfDay()) ?: false
-  }
 
   /**
-   * A standard move is a completed move with a single completed journey that is billable, and no cancelled journeys
-   * To be priced as a standard move, the journey as well as the move must be completed
-   * There also should be no redirects after the move starts, but shouldn't need to check for this
+   * A standard move is a completed move with a single completed journey that is billable, the journey from and to
+   * destinations should match that of the move, and no cancelled journeys. To be priced as a standard move, the journey
+   * as well as the move must be completed. There also should be no redirects after the move starts, but shouldn't need
+   * to check for this.
    */
-  fun isStandardMove(move: Move): Boolean {
-    return isCompleted(move) &&
-      with(move.journeys.map { it }) {
+  fun isStandardMove(move: Move) =
+    isCompleted(move) &&
+      with(move.journeys) {
         count { it.stateIsAnyOf(JourneyState.completed) } == 1 &&
-          count { it.stateIsAnyOf(JourneyState.completed) && it.billable } == 1 &&
+          count { isCompleteBillableJourneyAndLocationsMatchMove(it, move) } == 1 &&
```

Comment:
This is and the prive fun below are the key changes for this PR.  The other bits are some small tidy-ups/refactorings.